zerver/lib/actions.py

```diff
@@ -6004,9 +6004,9 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
                 subscriptions = subscriptions.exclude(
                     user_profile_id__in=[sub.user_profile_id for sub in old_stream_unsubbed_guests]
                 )
-                subscriber_ids = [user.user_profile_id for user in subscriptions]
+                subscriber_ids = set(subscriptions.values_list("user_profile_id", flat=True))
 
-            users_to_be_notified += list(map(subscriber_info, subscriber_ids))
+            users_to_be_notified += list(map(subscriber_info, sorted(list(subscriber_ids))))
```

Comment:
Why the sort?

Reply:
determinism for tests, and sort's super fast

Reply:
Ahh, determinism is a good reason.  Though this reminds me that we probably want to do the analog of
f7aab9c92cdbd465412bf1950b0e7be5ceba48bc for other events.